src/openrct2/network/network.cpp

```diff
@@ -307,6 +307,15 @@ bool Network::BeginServer(uint16 port, const char* address)
 	player->Group = 0;
 	player_id = player->Id;
 
+	if (network_get_mode() == NETWORK_MODE_SERVER) {
+		// Add SERVER to users.json and save.
+		NetworkUserManager *userManager = &gNetwork._userManager;
```

Comment:
Is there a point to getting the user manager *pointer*? `NetworkUserManager` is a `final` class and it is already a member of `Network`.